lib/native/source/opensmt/swigWrapper.patch

```diff
@@ -0,0 +1,416 @@
+diff --git a/lib/native/source/opensmt/opensmt-wrap.cpp b/lib/native/source/opensmt/opensmt-wrap.cpp
+index cfe601cd2..70df21180 100644
```

Comment:
This file is funny: a patch file for license headers, and it has a separate license file for itself.

Good idea and valid solution.

PS: You could also have created a separate license file `FILE.i.license` per source file `FILE.i`.